tests/results/scale/edge/edge-oss.md

```diff
@@ -179,22 +179,22 @@ The logs are attached only if there are errors.
 ## Test TestScale_HTTPMatches
 
 ```text
-Requests      [total, rate, throughput]         29999, 1000.00, 999.98
-Duration      [total, attack, wait]             30s, 29.999s, 606.451µs
-Latencies     [min, mean, 50, 90, 95, 99, max]  526.999µs, 683.711µs, 660.153µs, 741.92µs, 772.925µs, 867.538µs, 21.592ms
-Bytes In      [total, mean]                     4799840, 160.00
+Requests      [total, rate, throughput]         30000, 1000.01, 999.98
+Duration      [total, attack, wait]             30.001s, 30s, 795.282µs
+Latencies     [min, mean, 50, 90, 95, 99, max]  571.07µs, 775.874µs, 750.269µs, 856.654µs, 899.524µs, 1.032ms, 13.244ms
+Bytes In      [total, mean]                     4860000, 162.00
```

Comment:
I do see latencies increase by 3-4 seconds on max number of requests when scaling and this change was almost 8ms. 

Do we need to be concerned? What are the acceptable +- ms for these? @sjberman

Reply:
yeah I agree, ms is pretty small